tests/test_data/receptyprevas.sk/receptyprevas.json

```diff
@@ -0,0 +1,64 @@
+{
+  "host": "receptyprevas.sk",
+  "canonical_url": "https://www.receptyprevas.sk/recepty/buchty-na-pare-babickin-recept-na-parene-buchty/",
+  "site_name": "Recepty pre Vás",
+  "author": "Martina",
+  "language": "sk-SK",
+  "title": "Buchty na pare, babičkin RECEPT na parené buchty",
+  "ingredients": [
+    "Čerstvé droždie: 0.5 ks",
+    "Mlieko: 1 dl",
+    "Kryštálový cukor: 1 PL",
+    "Polohrubá múka: 500 g",
+    "Minerálka: 1 dl",
+    "Mlieko: 1 dl",
+    "Kryštálový cukor: 2 PL",
+    "Vajce: 1 ks",
+    "Olej: 2 PL",
+    "Soľ: 1 štipka",
+    "Lekvár: 16 ML",
```

Comment:
I think so, but to make sure: `ML != ml` ?

Reply:
here `ML` stands for "malá lyžička" = "tea spoon". i believe "čl" (čajová lyžička, note that "čaj" translates to tea) is used way more often.

Reply:
Ok, thank you!